### PR TITLE
Combined desktop dependency upgrades

### PR DESCRIPTION
**Pull request by jakubgs**
This batches up desktop package upgrades from PRs #9243 to #9252.

### CODE REVIEW

**Comment by ghost** [comment]
`desktop/js_files/yarn.lock` and `mobile/js_files/yarn.lock` changed. Pinging @jakubgs, @PombeirP, and @corpetty

**Comment by ghost** [comment]
`desktop/js_files/yarn.lock` and `mobile/js_files/yarn.lock` changed. Pinging @jakubgs, @PombeirP, and @corpetty

**Review by pedropombeiro — APPROVED** [review]

File: mobile/js_files/package.json [hunk 1]
```diff
@@ -33,6 +34,7 @@
     "react-native-languages": "^3.0.2",
     "react-native-mail": "git+https://github.com/status-im/react-native-mail.git#v4.0.0-status",
     "react-native-navigation-twopane": "git+https://github.com/status-im/react-native-navigation-twopane.git#v0.0.2-status",
+    "react-native-os": "^1.2.5",
```
Comment (pedropombeiro) [inline mobile/js_files/package.json]: @jakubgs why is `react-native-os` reappearing in the mobile `package.json`?

Comment (jakubgs) [reply mobile/js_files/package.json]: Good question, probably because it was created before that was removed. Let me fix that.

**Pull request merged**